tools/buildutils/cw/build_package.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2025 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+>&2 echo "Working dir: $(pwd)"
```

Comment:
can this share code with tools/buildutils/build_packages.sh?

Reply:
Those two lines have 10 command line arguments in total, I'd say it's very likely that those commands are modified further in the future, so I prefer to keep them in a single place.

Reply:
Got it! Extracted building single package command into its own script. PTAL!